docs/audio_driver.md

```diff
@@ -0,0 +1,122 @@
+# Audio Driver
+
+The [Audio feature](feature_audio.md) breaks the hardware specifics out into separate, exchangeable driver units, with a common interface to the audio-"core" - which itself handles playing songs and notes while tracking their progress in an internal state, initializing/starting/stopping the driver as needed.
+
+Not all MCUs support every available driver, either the platform-support is not there (yet?) or the MCU simply does not have the required hardware peripheral.
+
+
+## AVR
+
+Boards build around an Atmega32U4 can use two sets of PWM capable pins, each driving a separate speaker.
+The possible configurations are:
+
+|              | Timer3      | Timer1       |
+| one speaker  | C4,C5 or C6 |              |
```

Comment:
The markdown parser doesn't know to make this a table unless you include a line to separate the headings from the content.

```suggestion
|--------------|-------------|--------------|
| one speaker  | C4,C5 or C6 |              |
```